git/packfile.go

```diff
@@ -46,7 +46,7 @@ func (t PackEntryType) String() string {
 	case OBJ_REF_DELTA:
 		return "ref_delta"
 	default:
-		return "unknown"
+		return fmt.Sprintf("unknown: %x", uint8(t))
```

Comment:
not sure if you're keeping this or it was just for debugging it, but if you're keeping it it should probably have a "0x" prefix.